src/main/java/org/terasology/ligthandshadow/componentsystem/controllers/ClientGameOverSystem.java

```diff
@@ -86,19 +87,27 @@ public void onGameOver(GameOverEvent event, EntityRef entity) {
     }
 
     private void addPlayerStatisticsInfo(DeathScreen deathScreen) {
-        MigLayout migLayout = deathScreen.find("playerStatistics", MigLayout.class);
-        if (migLayout != null) {
+        MigLayout spadesTeamMigLayout = deathScreen.find("spadesTeamPlayerStatistics", MigLayout.class);
+        MigLayout heartsTeamMigLayout = deathScreen.find("heartsTeamPlayerStatistics", MigLayout.class);
+        MigLayout migLayout;
+        if (spadesTeamMigLayout != null && heartsTeamMigLayout != null) {
             Iterable<EntityRef> characters = entityManager.getEntitiesWith(PlayerCharacterComponent.class, LASTeamComponent.class);
             for (EntityRef character : characters) {
                 EntityRef client = character.getOwner();
                 ClientComponent clientComponent = client.getComponent(ClientComponent.class);
-                migLayout.addWidget(new UILabel(PlayerUtil.getColoredPlayerName(clientComponent.clientInfo)), new MigLayout.CCHint());
+                String playerTeam = localPlayer.getCharacterEntity().getComponent(LASTeamComponent.class).team;
                 PlayerStatisticsComponent playerStatisticsComponent = character.getComponent(PlayerStatisticsComponent.class);
-                migLayout.addWidget(new UILabel(String.valueOf(playerStatisticsComponent.kills)), new MigLayout.CCHint());
-                migLayout.addWidget(new UILabel(String.valueOf(playerStatisticsComponent.deaths)), new MigLayout.CCHint("wrap"));
+                migLayout = (playerTeam.equals("black") ? spadesTeamMigLayout : heartsTeamMigLayout);
+                addInfoToTeamMigLayout(migLayout,clientComponent,playerStatisticsComponent);
```

Comment:
I was in between two ways of implementing this:

1. have an extra migLayout variable, and hold there the migLayout that the playerDetails are gonna be added at: _(this is also what I am doing at the marked lines)_
```JAVA
migLayout = (playerTeam.equals("black") ? spadesTeamMigLayout : heartsTeamMigLayout);
addInfoToTeamMigLayout(migLayout,clientComponent,playerStatisticsComponent);
```
2. have an if statement and repeat the call of `addInfoToTeamMigLayout` twice, but with a different migLayout sent each time:
```JAVA
if (playerTeam.equals("black")){
     addInfoToTeamMigLayout(spadesTeamMigLayout, clientComponent, playerStatisticsComponent);
} else {
     addInfoToTeamMigLayout(heartsTeamMigLayout, clientComponent, playerStatisticsComponent);
}
```

With _1._ we avoid repeating the same code (the call of addInfoToTeamMigLayout) and the code looks more clear and simple, while with the _2._ we avoid having an extra variable. I believe the _1._ is a better option, thus why I went with it, but I want to hear your opinion as well @skaldarnar and @jdrueckert .